lcviewernoqt/drawitems/lcvhatch.cpp

```diff
@@ -10,11 +15,72 @@ LCVHatch::LCVHatch(const lc::entity::Hatch_CSPtr& hatch) :
 }
 
 void LCVHatch::draw(LcPainter& painter, const LcDrawOptions &options, const lc::geo::Area& rect) const {
-    painter.circle(0, 0, 20);
-    painter.fill();
+    for(auto &x:_hatch->_loopList){
+        bool processingLine = false; //To make polyline from lines for hatch
+        for(auto &y:x->objList){
+            if(auto arc = std::dynamic_pointer_cast<const lc::entity::Arc>(y)){
+                if (arc->radius() != 0) {
+		std::cout << arc->CCW() << 'x' <<arc->center().x() << 'y' <<arc->center().y() << 'r' <<arc->radius() << 's' <<arc->startAngle() << 'e' <<arc->endAngle() <<std::endl;
```

Comment:
You forgot one debug line here :)
You can lckernel's `logger.h` with `LOG_DEBUG` if you want to let the lines here

Reply:
ok